go-build-ldflags.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+PROJECT_PACKAGE=$0
```

Comment:
iirc $0 is the program name, should this be `$1`?

also, could you return error if its not set

Reply:
Ha, yes true.  Also can do re: return error if not set.